integration-test/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -46,8 +45,8 @@ private ContainerSuite() {
       Info info = dockerClient.infoCmd().exec();
       LOG.info("Docker info: {}", info);
 
-      if (System.getenv("NEED_CREATE_DOCKER_NETWORK") != null
-          && Objects.equals(System.getenv("NEED_CREATE_DOCKER_NETWORK"), "true")) {
+      String needCreateDockerNetwork = System.getenv("NEED_CREATE_DOCKER_NETWORK");
+      if (needCreateDockerNetwork != null && needCreateDockerNetwork.equalsIgnoreCase("true")) {
```

Comment:
Use `"true".equals(xxx)` instead and we can remove `needCreateDockerNetwork != null`

Reply:
I think it's ok for me. We can improve it later on. @xunliu do you want to fix it?